plugins/ingest-attachment/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -33,4 +33,6 @@ grant {
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   // PDFBox checks for the existence of this class
   permission java.lang.RuntimePermission "accessClassInPackage.sun.java2d.cmm.kcms";
+  // TODO: Remove once https://issues.apache.org/jira/browse/TIKA-2731 is resolved
```

Comment:
As we discussed via another thread, let’s hold here until 1.19.1 is released so that we do not need to make this grant.